cluster/cluster.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export KUBEVIRT_PROVIDER="${KUBEVIRT_PROVIDER:-k8s-1.23}"
+export KUBEVIRT_PROVIDER="${KUBEVIRT_PROVIDER:-k8s-1.24}"
```

Comment:
this make the PR title not just CI (ref to prefix)

Reply:
Done.